packages/shorebird_code_push_protocol/lib/src/messages/get_usage/get_usage_response.dart

```diff
@@ -26,14 +26,15 @@ class GetUsageResponse {
   /// The usage per app.
   final List<AppUsage> apps;
 
-  /// The start of the current billin period.
-  final DateTime? currentPeriodStart;
+  /// The start of the current billing period.
+  final DateTime currentPeriodStart;
 
-  /// The end of the current billin period.
-  final DateTime? currentPeriodEnd;
+  /// The end of the current billing period.
+  final DateTime currentPeriodEnd;
 
   /// The upper limit of patch installs for the current billing period.
-  final int patchInstallLimit;
+  /// If `null`, there is no limit.
+  final int? patchInstallLimit;
```

Comment:
Didn't catch this before, but what do you think about `patchDownloadLimit` instead, as that's what we're actually measuring?

Reply:
yup good call will update now.